examples/client_test.cpp

```diff
@@ -1558,6 +1558,8 @@ int main(int argc, char* argv[])
 		, listen_port);
 	settings.set_str(settings_pack::listen_interfaces, iface_str);
 
+	settings.set_int(settings_pack::urlseed_max_request_bytes, 32 * 1024 * 1024);
```

Comment:
I think it would be better to set the default to something sensible, than to set it explicitly here. in master, client_test provides generic access to all settings via the command line too